server/repository/originalDataPoint/update.ts

```diff
@@ -0,0 +1,52 @@
+import { BaseProtocol, DB, Schemas } from '@server/db'
+import { Assessment, Cycle, OriginalDataPoint } from '@meta/assessment'
+import { getOne } from './getOne'
+
+export const update = async (
+  params: { assessment: Assessment; assessmentCycle: Cycle; originalDataPoint: OriginalDataPoint },
+  client: BaseProtocol = DB
+): Promise<OriginalDataPoint | null> => {
+  const {
+    assessment,
+    assessmentCycle,
+    originalDataPoint: {
+      id,
+      countryIso,
+      year,
+      dataSourceAdditionalComments,
+      dataSourceMethods,
+      dataSourceReferences,
+      description,
+      nationalClasses,
+    },
+  } = params
+
+  const schemaName = Schemas.getNameCycle(assessment, assessmentCycle)
+
+  await client.oneOrNone<OriginalDataPoint | null>(
```

Comment:
why can this return null ? if we try to update an odp that doesn't exist, we should get back an exception